requre/utils.py

```diff
@@ -140,6 +141,32 @@ def replace(obj: Any, key: Any, value: Any) -> None:
             for item in obj:
                 DictProcessing.replace(obj=item, key=key, value=value)
 
+    def simplify(
+        self, internal_object: Optional[Dict] = None, ignore_list: Optional[List] = None
+    ):
+        if ignore_list is None:
+            ignore_list = []
+        if internal_object is None:
+            internal_object = self.requre_dict
```

Comment:
```python
internal_object = internal_object or self.requre_dict
```

Reply:
@lachmanfrantisek it will not work as expected, or better to say, I have to check ``None`` it will also match ``{}`` what could cause side effect. 
so possible to rewrite to ``internal_object = self.requre_dict if internal_object is None else iternal_object`` not sure if this is more readable

Reply:
OK then.